activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/changelog/h2.schema.sql

```diff
@@ -86,6 +86,9 @@ create table process_instance
     start_date                 timestamp,
     start_from                 timestamp,
     start_to                   timestamp,
+    completed_date             timestamp,
+    completed_from             timestamp,
+    completed_to               timestamp,
```

Comment:
`completed_from` and `completed_to` should not be part of the schema.

Reply:
Removed those transient parameter options.

Reply:
I see that upon removal of these params the tests are now failing, when I revert it (ie. add it back) test works fine.

Reply:
Fixed!